### DISCUSSION TITLE
DietPi-Software | Rockstor NAS

### DISCUSSION THREAD

**Discussion by AnthGood** [MichaIng/DietPi#5940] [Software Request]
I would absolutely love it if Dietpi offered support for the Rockstor NAS.

I had switched over to it from OMV a year ago using my x86 machine. A year ago they offered Raspberry Pi support (and I believe support for other ARM devices if you can compile it yourself) I do not know how to do this so I was hoping if DietPi supported it this would make installation way easier. A lot of people use their Raspberry Pis as NAS devices so having support for this would be helpeful.

https://rockstor.com/dls.html

**Comment by Joulinar** [comment]
Question: You are able to boot/install plain RPI OS 64bit image on your Rockstor NAS?

**Reply by AnthGood** [reply]
Sorry, I honestly don't know.

**Reply by Joulinar** [reply]
ah sorry I misunderstood. Rockstor NAS is not a device, it is a software stack like OMV.

**Reply by AnthGood** [reply]
Sorry, reading back my initial post it can be read that the Rockstor NAS is a device I wanted Diet Pi onto. I meant the other way, around I have a Rasperbery Pi and wanted an easy way to get Rockstor NAS onto it using DietPI Os.
Like you say its just a competitor NAS software to OMV. I just prefer it to OMV

**Comment by MichaIng** [comment]
This is not the kind of software we implement into DietPi. There seems to be a way to install it on vanilla OpenSUSE, but otherwise full disk images are provided: https://rockstor.com/dls.html
Even if we did manage to find a way to install into a DietPi system, it transforms the system so massively that there is nothing left of DietPi, but our scripts only conflicting with what Rockstor did/requires (NetworkManager, to mention one).

Not sure what benefit you hope for when installing it onto DietPi instead of flashing the Rockstor images directly. The latter is fully supported by them, you are sure that everything works etc. The first, as far as I can see, only causes a lot of development time and ongoing maintenance, and limitation on both: DietPi and Rockstor features.
